src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerIntegrationTest.java

```diff
@@ -71,6 +73,30 @@ public void testVCFModeIsConsistentWithPastResults(final String inputFileName, f
         IntegrationTestSpec.assertEqualTextFiles(output, expected);
     }
 
+    @Test
+    public void testSitesOnlyMode() {
```

Comment:
Move this test after the standard `testVCFMode*` and `testGVCFMode*` tests, rather than sticking it in the middle of them.

Reply:
Also call `Utils.resetRandomGenerator()` at the beginning of the test.

Reply:
done